scipy/interpolate/fitpack2.py

```diff
@@ -377,7 +378,7 @@ def get_coeffs(self):
         return data[9][:n-k-1]
 
     def get_residual(self):
-        """Return weighted sum of squared residuals of the spline approximation.
+        """ Return weighted sum of squared residuals of the spline approximation.
```

Comment:
It should be without the space actually.

Reply:
I agree